### ISSUE TITLE
Issues with waydroid

### ISSUE THREAD

**Issue by Quackdoc** [pop-os/cosmic-comp#962]
* On my laptop, Waydroid currently starts with the bottom part started off screen due to the taskbar. This was an issue with `xdg_shell set_maximized` on sway, not sure if it shares the same issue.

* On my desktop waydroid starts in the top left corner, and takes about the full quadrant of the screen. plan to investigate this more

**Comment by CodePolygon** [comment]
yeah, it still appears in top left corner of screen, cant resize it. and it also doesnt respect cosmic tiling.

**Comment by Quackdoc** [comment]
closed by: b2ba402e2dc014cd3b6fe2b8cb4a68aac3099c94
